base_local_planner/src/speed_limiters/static_object_speed_limiter.cpp

```diff
@@ -231,7 +231,7 @@ Chuck will cycle between speeding up and slowing down.
         currLinearVel <= params_.max_linear_velocity_test_speed)
     {
       SpeedLimiterResult result =
-          calculateAllowedLinearSpeed(distance_from_static_left, distance_from_static_right, max_allowed_linear_vel);
+          calculateAllowedLinearSpeed(distance_from_static_left, distance_from_static_right, velocity_.linear);
```

Comment:
nothing major.. just curious what is the logic behind this change..

Reply:
This was actually a bug in the sped limiter. It is supposed to reduce current speed and not the max speed, since it still might not be safe to avoid the ce trip.  I brought this up to Billy and said he was fine with me tossing this in this change instead of creating a new ticket